SUMMARY.md

```diff
@@ -13,11 +13,14 @@
 * [Clustered Wallaroo](book/core-concepts/clustering.md)
 
 ## Developing with Wallaroo
-
-* [Setting up Your Environment](book/getting-started/setup.md)
-  * [MacOS Installation](book/getting-started/macos-setup.md)
-  * [Ubuntu Installation](book/getting-started/linux-setup.md)
-* [Run a Wallaroo Application](book/getting-started/run-a-wallaroo-application.md)
+* First Install
```

Comment:
I think that "First Install" might be a confusing term for this. If the user doesn't have the background for what's happening here they may not understand why there is a "first" install, if there will be other installs (second, third, etc), what this install involves, or why it is a step at all.

I'm not sure how to best address this. Part of me thinks that maybe this section should be called something like "Using the Docker Quickstart Environment" or something like that. I don't really feel great about that title, but I think it does a better job of expressing the intention here.

Reply:
Installation or Getting Started is much better than "First Install"